.nvmrc

```diff
@@ -1 +1 @@
-v8.11.3
\ No newline at end of file
+v12.14.1
```

Comment:
`npx tslint-to-eslint-config` needed a new version of Node, so I picked this one.